pyproject.toml

```diff
@@ -9,11 +9,11 @@ packages = [{include = "gardenlinux", from="src"}, {include = "python_gardenlinu
 
 [tool.poetry.dependencies]
 python = "^3.10"
+poetry = "^2.1.3"
```

Comment:
I'm not sure if "poetry" can depend on itself here.

Reply:
Actually I tried this to bootstrap poetry... But this will not work anyways as the lock file is not yet in `main`.
I can remove this. The current alternative is that the `Makefile` now installs poetry if it is not found.